interface.html

```diff
@@ -52,6 +55,38 @@
             <span class="text-helper">Users will use the password on this field to log in.</span>
           </div>
         </div>
+        <h3></h3>
+        <div class="form-group clearfix">
+          <div class="col-sm-4 control-label">
+            <label for="logged-in-user-time">Keep the user logged in for
+              <a class="help-icon" data-toggle="tooltip" data-placement="top" title="Force user re-authentication after a number of hours">
+                <i class="fa fa-question-circle-o"></i>
+              </a>
+            </label>
+          </div>
+          <div class="col-sm-8">
+            <div class="input-group">
+              <input type="number" class="form-control logged-user" name="loggedInUserTime" id="logged-in-user-time" min="0" oninput="validity.valid||(value='');" value="{{sessionMaxDurationMinutes}}">
```

Comment:
What's the `name` attribute used for? It's not used as a selector because `id` is used for that.

```suggestion
              <input type="number" class="form-control logged-user" id="logged-in-user-time" min="0" oninput="validity.valid||(value='');" value="{{sessionMaxDurationMinutes}}">
```